### ISSUE TITLE
Embedding a video on the site

### ISSUE THREAD

**Issue by robgpeak** [robgpeak/CompLinks-xclub#135]
@cgladue please have a look at https://shop.complinks.co/How-it-works to see what I was trying to accomplish.  Ideally I would like to have the video show up in a modal automatically when the page loads, and then to have a button in that first section that would relaunch that same modal with the video in it in case someone closes it, they can reopen it.

I might need to resize the embedded video player, but that's fine...  Here is the video. https://youtu.be/cfwpJ_qJ4pc

**Comment by cgladue** [comment]
ok, it seems like something that Chris could do ... personally i cant stand websites that force me to wactch a video as soon as it loads ... are you set on that ?

**Comment by cgladue** [comment]
@robgpeak just throwing it out there, what about something like this to help walk the user through how to use the site ...

https://introjs.com/  scroll down to the "Examples" section

**Comment by robgpeak** [comment]
I would not put it on the home page, but I think it would not be that out of context to have the video load in a modal when someone gets to the how it works page.  But I get your point.  If someone comse back to the page for whatever reason, it would be annoying to see the pop up for the same video again.

So how about just (for now), being able to embed a video in that top section of the how it works page?

**Comment by robgpeak** [comment]
@cgladue do either of these help?

https://www.extro.media/support-de/faq/136-tinymce-using-the-script-tag-or-embed-youtube-video-with-iframe.html

https://www.tinymce.com/docs/plugins/media/

**Comment by robgpeak** [comment]
To be clear, I don't need to to pop open as a modal, it can be embeded on the page where I have the code already in the top section on the page https://shop.complinks.co/How-it-works

**Comment by cgladue** [comment]
do you know what size you want it to be ?

**Comment by robgpeak** [comment]
not really, but it needs to be "flexible" based on the size of someone's screen.  So I am not sure how Intercom does it, but if you look at the video on the page https://support.complinks.co/how-to-video-collection/how-to-view-store-deals-and-start-a-shopping-trip and then make your screen smaller to emulate the size of a phone screen, the container for the video gets smaller.

**Comment by robgpeak** [comment]
responsive.  That is the word I was looking for not flexible.

**Comment by cgladue** [comment]
https://shop.complinks.co/how-it-works

**Comment by cgladue** [comment]
@robgpeak  ?

**Comment by robgpeak** [comment]
No way to make it responsive? It shows up too big in the mobile view.
![screenshot_2018-05-09-15-47-35](https://user-images.githubusercontent.com/28393235/39836170-7c754c84-53a0-11e8-8482-96eab36b57d3.png)

**Comment by cgladue** [comment]
yea im sure @ChrisASearles would know how to, i dont know what CSS sorcesry is needed to make it shrink when the div shrinks ...

**Comment by robgpeak** [comment]
https://coolestguidesontheplanet.com/videodrome/youtube/  Here is the code that would need to be added to the css and then the div would be a `<div class="video-container">`

**Comment by cgladue** [comment]
i tried that already, and it dosnt work with the structure of our page,

**Comment by ChrisASearles** [comment]
I committed a responsive version of this but you're not going to like it. The addition of a responsive video in those sections really changes the way the whole page should be designed. Take a look and let me know if you want us to try to come up with a better alternative.

**Comment by cgladue** [comment]
@robgpeak published
